### PR TITLE
Governance document

### PR DESCRIPTION
**Pull request by lparsons**
The agreed upon draft of the US-RSE Governance document in Markdown format.

### CODE REVIEW

File: governance.md
Comment (cosden) [inline governance.md]: How about we just pick today? 
```suggestion
Date: June 12, 2020
```

**Review by JeffCarver — COMMENTED** [review]
Looks good to me.

File: governance.md
Comment (cosden) [inline governance.md]: It's a little bit harder here, but we still need an official date

```suggestion
This document was ratified by the following members of the First US-RSE Steering Committee on June
12, 2020:
```

File: governance.md
Comment (danielskatz) [inline governance.md]: should we list our names here?

File: governance.md
Comment (danielskatz) [inline governance.md]:
```suggestion
Membership is open to anyone who supports the mission of the US-RSE Association and who agrees to abide by
```

File: governance.md
Comment (danielskatz) [inline governance.md]:
```suggestion
all Committee meetings, a quorum is at least one half of the members.
```

File: governance.md
Comment (danielskatz) [inline governance.md]: I'm not sure that we should specify bi-monthly, as at some point, we might want this to be monthly or even weekly.  Maybe we say "that a newsletters is sent out at least bi-monthly" or something even more general?

File: governance.md
Comment (danielskatz) [inline governance.md]:
```suggestion
Vice-Chair will annually prepare a report of Association activities in the past year.
```

File: governance.md
Comment (danielskatz) [inline governance.md]:
```suggestion
consist of one or more members of the Steering Committee and interested members of and participants in
```

File: governance.md
Comment (danielskatz) [inline governance.md]:
```suggestion
steering committee members. Major changes to the governance structure or this document will require
```

File: governance.md
Comment (danielskatz) [inline governance.md]:
```suggestion
Minor revisions to this document can be made with the approval of at least two thirds (⅔) of the current
```

File: governance.md
Comment (danielskatz) [inline governance.md]: election chair?

File: governance.md
Comment (cmaimone) [reply governance.md]: maybe that a newsletter will be sent out at least 3? times per year

File: governance.md
Comment (cmaimone) [inline governance.md]:
```suggestion
Minor revisions to this document can be made with the approval of at least two-thirds of the current
```

File: governance.md
Comment (cmaimone) [inline governance.md]:
```suggestion
* Christina Maimone (election chair)
```

**Review by cmaimone — APPROVED** [review]
Assuming changes identified by others are incorporated, I approve.

**Review by sandragesing — APPROVED** [review]
I'm fine with suggested changes and approve.

File: governance.md
Comment (cferenba) [reply governance.md]: or just "will be sent out regularly"?

File: governance.md
Comment (cferenba) [reply governance.md]: Will there be any technical issue with using a non-ASCII character here?

**Comment by cferenba** [comment]
I approve once the suggested changes are added/addressed.

**Review by cferenba — APPROVED** [review]

File: governance.md
Comment (lparsons) [reply governance.md]: I wouldn't think so these days.

File: governance.md
Comment (lparsons) [inline governance.md]:
```suggestion
Minor revisions to this document can be made with the approval of at least two-thirds (⅔) of the current
```
per @cmaimone

File: governance.md
Comment (lparsons) [reply governance.md]: Perhaps the date we get everyone to approve?

File: governance.md
Comment (lparsons) [reply governance.md]: That seems fine by me.

File: governance.md
Comment (jordanperr) [reply governance.md]: I suggest we use the date this is merged into the repo. Whoever does the merge should just make one final commit to update this date.

File: governance.md
Comment (jordanperr) [reply governance.md]: I suggest we use the date this is merged into the repo. Whoever does the merge should just make one final commit to update this date.

File: governance.md
Comment (jordanperr) [reply governance.md]: Having a list of authors could become cumbersome over time, especially since we've allowed revisions to this document. Is there a need to have an author list?

File: governance.md
Comment (jordanperr) [reply governance.md]: This date would be updated every time there is a merge.

File: governance.md
Comment (danielskatz) [reply governance.md]: If you want to just remove the line "Authors: The founding members of the US-RSE Association Steering Committee", that would be fine with me

File: governance.md
Comment (danielskatz) [reply governance.md]: would once every 2 years be ok?  That would be regular.

File: governance.md
Comment (jordanperr) [inline governance.md:79]: I think it is important that we make an active effort to ensure "all" members are aware of the nomination process and how to participate. I had suggested putting in some more process details here - such as "an email will be sent out to all members informing them of the nomination procedures." I am fine keeping the wording as-is, but I want to emphasize the importance of making an effort to actively solicit the nominations from "all" members! In addition, the same active efforts should be made to boost participation in voting!

File: governance.md
Comment (jordanperr) [inline governance.md]: In addition to the author list, I think having such a "signatures" section may become cumbersome over time. What happens if a change is made down the road after one or more of us are no longer participating?

As this isn't a declaration of independence, I'm not sure what the value is of having our "signatures" here. Thoughts?

Comment (danielskatz) [reply governance.md]: it's fine with me to remove it

**Review by jordanperr — APPROVED** [review]
Thank you @lparsons for really moving this over the finish line!

Just some minor comments.

- One about wording in the nomination process, which I brought up at the previous meeting.
- One about the value of including an author list (or even the signatures).
- I've also made some suggestions on which dates to use.

Overall this looks great to me! I approve as-is and agree with the changes suggested so far. Very happy to have had the opportunity to help out with this and I look forward to the future!

**Comment by christophernhill** [comment]
Agreed - I think it can just say "by the US-RSE Steering Committee"

**Review by JeffCarver — APPROVED** [review]

File: governance.md
Comment (cosden) [reply governance.md]: OK. I like the suggestion that whoever merges updates the date.

File: governance.md
Comment (cosden) [reply governance.md]: Now that you bring it up, I agree that it could get cumbersome. I'm fine with removing the "Authors: .." line too.

File: governance.md
Comment (cosden) [reply governance.md]: For a governance document, I'm not sure we even need to specify newsletter responsibility. Is it critical to include? It could very easily become someone else's responsibility, a community coordinator/manager or committee. All that said, if it says in, I'd be fine with some form of regular rather than bi-monthly.

File: governance.md
Comment (cosden) [reply governance.md:79]: I agree with you, @jordanperr, we'll want to make lots of noise. I feel like we aren't a very "loud" organization, but leading up to the nomination/election we should be more persistent with emails, slack, etc.

File: governance.md
Comment (cosden) [reply governance.md]: I liked the signature section before this moved to github. The approvers will be clear so I think it's ok to remove it, but it's possible there is a good reason to have it in there. We should get agreement from everyone (including those who have already approved).

File: governance.md
Comment (lparsons) [reply governance.md]: I agree with Ian, I don't think that needs to specified in a governance document. This iteration of the Steering Committee likes doing it bi-monthly, later iterations may choose monthly, or semi-annual, or even just to eliminate it in favor of more frequent blog/twitter/tiktok/whatever posts. I'm in favor of removing this entirely or if we must, include something like: "The Vice-Chair will be responsible for ensuring the community is kept informed of Association activities through regular newsletters, social media posts, etc."

File: governance.md
Comment (lparsons) [reply governance.md]: I'm fine with either keeping it or removing it. It's probably simpler to remove.

Comment (lparsons) [reply governance.md]: If we do remove it, we will need to add the positions that people have taken, (chair, vice-chair, election chair, and secretary) via the transition letter, and imho, a page on the website (perhaps with short bios).

File: governance.md
Comment (cferenba) [reply governance.md]: Either of Lance's suggestions sounds fine to me.

File: governance.md
Comment (jordanperr) [reply governance.md]: There is already a page on the website. I suggest we simply keep the page up to date with the results of each election. It would also be nice if we keep a list of "former members of the steering committee" on that same web page - or easily accessible from that page.

https://us-rse.org/steering-committee/

File: governance.md
Comment (cferenba) [reply governance.md:79]: I agree we should make plenty of effort to notify everyone, but I don't think this document is the right place for specifics of how we do that.  I suggest we leave the wording as it is, but use this PR discussion as a note to make sure we have a good announcement system.

File: governance.md
Comment (danielskatz) [reply governance.md]: the last suggestion is ok with me

File: governance.md
Comment (danielskatz) [reply governance.md:79]: I also don't think we need to say how in this document

File: governance.md
Comment (danielskatz) [reply governance.md]: agreed

File: governance.md
Comment (cosden) [reply governance.md]: I'm good with Lance's suggestion too.

File: governance.md
Comment (cosden) [reply governance.md]: Sounds good. Agree that we should put the roles on the webpage. I added chair months ago before we had the others.

File: governance.md
Comment (sandragesing) [reply governance.md]: I also like Lance's suggestion.

Comment (lparsons) [reply governance.md]: OK, I've updated the document.

File: governance.md
Comment (lparsons) [reply governance.md:79]: I agree to leave as is, esp. with the added clarification by Jordan in the announcement letter.

**Review by cosden — APPROVED** [review]

**Review by danielskatz — APPROVED** [review]

**Review by christophernhill — APPROVED** [review]
All looks excellent to me - great job.

File: governance.md
Comment (lparsons) [inline governance.md]:
```suggestion
Date: June 29, 2020
```

File: governance.md
Comment (lparsons) [inline governance.md]:
```suggestion
This document was ratified by the following members of the First US-RSE Steering Committee on June
29, 2020.
```

**Pull request merged**
